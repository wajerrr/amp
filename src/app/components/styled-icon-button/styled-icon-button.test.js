import React from 'react';
import renderer from 'react-test-renderer';
import StyledIconButton from './styled-icon-button';

describe('StyledIconButton', () => {
  it('should match snapshot', () => {
    const tree = renderer
      .create(<StyledIconButton onProps="toggleMenu" icon="hambuger" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
