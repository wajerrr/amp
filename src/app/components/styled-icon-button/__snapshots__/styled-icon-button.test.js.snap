// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledIconButton should match snapshot 1`] = `
<button
  aria-label=""
  className="css-1s236yn-StyledButton e139jkne0"
  on="toggleMenu"
  tabIndex="0"
/>
`;
